stdlib/Markdown/docs/src/index.md

```diff
@@ -47,13 +47,15 @@ parts of a Julia program.
 
 ### ``\LaTeX``
 
-Surround text that should be displayed as mathematics using ``\LaTeX`` syntax with double backticks,
-``` `` ``` .
+Surround text that should be displayed as mathematics using ``\LaTeX`` syntax with double backticks:
+``` `` ```.
 
 ```
-A paragraph containing some ``\LaTeX`` markup.
+A paragraph containing some ``\LaTeX`` markup. More $\LaTeX$ markup.
```

Comment:
I wouldn't add this here, since this to me sounds like it endorses the `$` syntax, which we shouldn't do.

```suggestion
A paragraph containing some ``\LaTeX`` markup.
```